common/traits/wc_professions_traits.txt

```diff
@@ -0,0 +1,464 @@
+﻿profession_mining = {
+	name = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_profession_mining
+			}
+			desc = trait_profession_mining_character
+		}
+	}
+	icon = profession_mining
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_profession_mining_desc
+			}
+			desc = trait_profession_mining_character_desc
+		}
+	}
+	shown_in_ruler_designer = no
+	minimum_age = 14
+
+	group = profession
+	level = 1
+	flag = profession_trait
+	flag = profession_mining
+	flag = profession_type_foraging
+	category = lifestyle
+	
+	same_opinion = 10
+
+	prowess = 3
+
+	track = {
+		20 = { }
+		40 = { }
+		60 = { }
+		80 = { }
+		100 = { }
+	}
+
+	compatibility = {
+		profession_blacksmithing = @pos_compat_high
+	}
+}
+
+profession_blacksmithing = {
+	icon = profession_blacksmithing
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_profession_blacksmithing_desc
+			}
+			desc = trait_profession_blacksmithing_character_desc
+		}
+	}
+	shown_in_ruler_designer = no
+	minimum_age = 14
+
+	group = profession
+	level = 1
+	flag = profession_trait
+	flag = profession_blacksmithing
+	flag = profession_type_craft
+	category = lifestyle
+
+	same_opinion = 10
+
+	prowess = 3
+
+	track = {
+		20 = { }
+		40 = { }
+		60 = { }
+		80 = { }
+		100 = { }
+	}
+
+	compatibility = {
+		profession_mining = @pos_compat_high
+	}
+}
+
+profession_skinning = {
+	icon = profession_skinning
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_profession_skinning_desc
+			}
+			desc = trait_profession_skinning_character_desc
+		}
+	}
+	shown_in_ruler_designer = no
+	minimum_age = 14
+
+	group = profession
+	level = 1
+	flag = profession_trait
+	flag = profession_skinning
+	category = lifestyle
+
+	same_opinion = 10
+
+
+
+	track = {
+		20 = { }
+		40 = { }
+		60 = { }
+		80 = { }
+		100 = { }
+	}
+
+	compatibility = {
+		profession_leatherworking = @pos_compat_high
+	}
+}
+
+profession_herbalism = {
+	icon = profession_herbalism
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_profession_herbalism_desc
+			}
+			desc = trait_profession_herbalism_character_desc
+		}
+	}
+	shown_in_ruler_designer = no
+	minimum_age = 14
+
+	group = profession
+	level = 1
+	flag = profession_trait
+	flag = profession_skinning
+	category = lifestyle
+
+	same_opinion = 10
+
+
+
+	track = {
+		20 = { }
+		40 = { }
+		60 = { }
+		80 = { }
+		100 = { }
+	}
+
+	compatibility = {
+		profession_alchemy = @pos_compat_high
+	}
+}
```

Comment:
Do you think the already existing herbalist lifestyle could be used for that instead? 
Like taking all the flags, tracks and anything else you want to use from here and putting them into `lifestyle_herbalist`.
Because having both traits could be confusing.

Reply:
i am consiente this yes, but I'm gonna work on other professions first and then probably redo the herbalist lifestyle